app/views/provider_user_mailer/activation_reminder.text.erb

```diff
@@ -1,10 +1,11 @@
 <%= @user.informal_name %>,
 
-A couple of days ago you signed up for <%= master_mailer_name %> to manage your API.
+A couple of days ago you signed up for Red Hat 3scale to manage your API.
 Now would be as good a time as any to activate your account:
 
 <%= @activate_url %>
 
-Thanks!
+You will then be able to login with your email and password. If you have problems activating your account please contact customer service (https://access.redhat.com/customerservice).
 
-The <%= master_mailer_name %> team.
+Cheers,
```

Comment:
we should keep on using the method, otherwise all on prem master accounts will have this signature. The fallback for saas in that method should be updated: https://github.com/3scale/porta/blob/THREESCALE-1883/app/helpers/mailers_helper.rb#L4

Reply:
I put it back